invoicing.go

```diff
@@ -0,0 +1,38 @@
+package paypal
+
+import (
+	"context"
+	"fmt"
+)
+
+// GenerateInvoiceNumber: generates the next invoice number that is available to the merchant.
+// Endpoint: POST /v2/invoicing/generate-next-invoice-number
+func (c *Client) GenerateInvoiceNumber(ctx context.Context) (*InvoiceNumber, error) {
```

Comment:
Please also add that to README to the list of supported enspoints.

Reply:
Added in [this commit](https://github.com/plutov/paypal/pull/249/commits/170473c605267a17aa86594060071130adae08ea), please review.